### DISCUSSION TITLE
v3.15.0

### DISCUSSION THREAD

**Discussion by rodvand** [netbox-community/ansible_modules#1100] [General]
# Minor changes
* Fix device_role -> role and add testing for Netbox 3.6 by @FragmentedPacket in https://github.com/netbox-community/ansible_modules/pull/1066
* Bump cryptography from 41.0.3 to 41.0.4 by @dependabot in https://github.com/netbox-community/ansible_modules/pull/1072
* Bump urllib3 from 2.0.4 to 2.0.6 by @dependabot in https://github.com/netbox-community/ansible_modules/pull/1079
* feat: inventory item component by @ahlee1 in https://github.com/netbox-community/ansible_modules/pull/1078
* Bump urllib3 from 2.0.6 to 2.0.7 by @dependabot in https://github.com/netbox-community/ansible_modules/pull/1094
* Fix CI test and update sphinx configuration by @rodvand in https://github.com/netbox-community/ansible_modules/pull/1096
* feat: Add support for default_platform on device_types by @MinDBreaK in https://github.com/netbox-community/ansible_modules/pull/1092
* feat: Add config-template support by @MinDBreaK in https://github.com/netbox-community/ansible_modules/pull/1090

## New Contributors
* @ahlee1 made their first contribution in https://github.com/netbox-community/ansible_modules/pull/1078

**Full Changelog**: https://github.com/netbox-community/ansible_modules/compare/v3.14.0...v3.15.0

<hr /><em>This discussion was created from the release <a href='https://github.com/netbox-community/ansible_modules/releases/tag/v3.15.0'>v3.15.0</a>.</em>

**Comment by shaderecker** [comment]
A big thanks for the new release! 🎉 I really appreciate it.
I was wondering how the new version will land on Ansible Galaxy (https://galaxy.ansible.com/ui/repo/published/netbox/netbox/)?

**Reply by MinDBreaK** [reply]
Looks like the deploy failed, maybe the migration to the new galaxy broke some APIs :/

**Reply by rodvand** [reply]
Correct - we're looking into it.

**Reply by shaderecker** [reply]
Ok, thank you for taking care ❤️

**Comment by rodvand** [comment]
The new version is now on ansible galaxy.

**Reply by shaderecker** [reply]
Nice 🥳 many thanks @rodvand!
